src/components/Layout.tsx

```diff
@@ -52,7 +52,6 @@ export interface RowProps {
 export const FlexRow = styled.div<RowProps>`
   display: flex;
   flex-direction: row;
-  flex-wrap: wrap;
```

Comment:
Har du sjekket at dette er trygt å gjøre? Ettersom FlexRow er generisk og kan brukes mange steder.

Reply:
Ut i fra det jeg har testet, så virker det som om det har gått ganske greit. Det er relativt enkelt å reversere hvis det roter til ting da. 

Eller tror du det er mer hensiktsmessig å lage ny komponent for denne casen?

Reply:
Når endring gjøres, må man sjekke hvilke andre steder av kodene som påvirkes av dette og hvordan det evt. påvirker eksisterende kode.
Ut ifra et kjapt søk så finner jeg rundt 40 steder for <FlexRow> brukes. En slik endring vil da endre opptil 40 andre steder.
Enten så må man vite at denne endringen går greit for de 40 stedene den er i bruk eller så må man ikke gjøre en endring som endrer logikk for alle som bruker <FlexRow> og heller gjøre en endring som kun påvirker InfoRow.tsx.
Med Styled-components  kan dette gjøres både med å overstyre en property når man utvider en eksisterende komponent, eller ved bruk props slik at man får flere alternative varianter og kan ha en default(i dette tilfelle "wrap").